JS-do-not-construct-regex-from-userinput/hjp007_musicShare_router.js

```
@@ -0,0 +1,342 @@
+
+var User = mongoose.model("User"); 
+var Song = mongoose.model("Song"); 
+var FriendRequest = mongoose.model("FriendRequest"); 
+var ShareRequest = mongoose.model("ShareRequest"); 
+
+var route = express.Router();
+route.post('/test', function(req,res,next){
+    co(function*(){
+        logger.info(req.body);
+        return Config.SUCCESS(req.body.test);
+    }).then(function(data){
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)